### PR TITLE
Updated info and configuration for easier enterprise quick start doc

### PR DESCRIPTION
**Pull request by stevenGravy**
- updated to use specific node names to match auth and node names instead of default hostnames.
 - modified language that was heavily ssh centric
 - showed how to use ca_pin from auth service
 - added info on checking the status of the teleport and other helpful info

### CODE REVIEW

File: docs/5.0/pages/enterprise/quickstart-enterprise.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ and `auth servers`.
 - **Proxy servers** route client connection requests to the appropriate node and serve a
   Web UI which can also be used to log into SSH nodes. Every client-to-node
   connection in Teleport must be routed via a proxy.
-- **Nodes** are regular SSH servers, similar to the `sshd` daemon you may be familiar
+- **Nodes** provide access to resources including ssh, kubernetes, web applications
```
Comment (webvictim) [inline docs/5.0/pages/enterprise/quickstart-enterprise.md:20]:
```suggestion
- **Nodes** provide access to resources including SSH, Kubernetes, web applications
```

File: docs/5.0/pages/enterprise/quickstart-enterprise.md [hunk 2]
```diff
@@ -17,7 +17,9 @@ and `auth servers`.
...
+  and databases.  In this quick start guide we are highlighting the ssh service. This ssh service
```
Comment (webvictim) [inline docs/5.0/pages/enterprise/quickstart-enterprise.md:21]:
```suggestion
  and databases.  In this quick start guide we are highlighting the SSH service. This SSH service
```

File: docs/5.0/pages/enterprise/quickstart-enterprise.md [hunk 3]
```diff
@@ -73,34 +75,31 @@ private key in [PEM format](https://en.wikipedia.org/wiki/Privacy-enhanced_Elect
...
+```bsh
+# run this on both servers:
+$ sudo systemctl daemon-reload
+$ sudo systemctl enable teleport
 ```
 
-Now, save the following configuration file as `/etc/teleport.yaml` on the _auth.example.com_:
+### Configuration File
+
+save the following configuration file as `/etc/teleport.yaml` on the _auth.example.com_:
```
Comment (webvictim) [inline docs/5.0/pages/enterprise/quickstart-enterprise.md:91]:
```suggestion
Save the following configuration file as `/etc/teleport.yaml` on the _auth.example.com_:
```

File: docs/5.0/pages/enterprise/quickstart-enterprise.md [hunk 4]
```diff
@@ -120,27 +119,65 @@ ssh_service:
...
+  auth_token: dogs-are-much-nicer-than-cats
+  # you can also use auth server's IP, i.e. "10.1.1.10:3025"
+  auth_servers: [ "auth.example.com:3025" ]
+  # Set the ca_pin value so it can verify the Teleport server
+  ca_pin: sha256:58b1c5cb8885d84ce08f3e...
+
+  # enable ssh service and disable auth and proxy:
+ssh_service:
+  enabled: true
+
```
Comment (webvictim) [inline docs/5.0/pages/enterprise/quickstart-enterprise.md:158]:
```suggestion
```

File: docs/5.0/pages/enterprise/quickstart-enterprise.md [hunk 5]
```diff
@@ -120,27 +119,65 @@ ssh_service:
...
 ```
+## Teleport Status and Networking
+
+You can review the logs of the Teleport service with `journalctl -fu teleport` and see its current status
+with `sudo systemctl status teleport`.
 
-Teleport daemon should start and you can use `netstat -lptne` to make sure that
-it's listening on [TCP/IP ports](../admin-guide.md#ports). On _auth.example.com_, it should
+You can use `netstat -lptne` to review the port
+listening on [TCP/IP ports](../admin-guide.md#ports). On _auth.example.com_, it should
```
Comment (webvictim) [inline docs/5.0/pages/enterprise/quickstart-enterprise.md:180]:
```suggestion
You can use `netstat -lptne` to review the port that Teleport is
listening on - see [TCP/IP ports](../admin-guide.md#ports). On _auth.example.com_, it should
```

**Comment by stevenGravy** [comment]
@webvictim applied ur suggestions, thanks!

**Review by webvictim — APPROVED** [review]
LGTM

File: docs/5.0/pages/enterprise/quickstart-enterprise.md [hunk 6]
```diff
@@ -39,11 +41,11 @@ the following DNS names and IPs:
 
 Server Name    |  IP Address    | Purpose
 ---------------|----------------|--------------
-_"auth.example.com"_  | 10.1.1.10      | This server will be used to run all three Teleport services: auth, proxy and node.
-_"node.example.com"_  | 10.1.1.11      | This server will only run the SSH service. The vast majority of servers in production will be nodes.
+_"auth.example.com"_  | 10.1.1.10      | This server will be used to run all three Teleport services: auth, proxy and ssh.
```
Comment (travelton) [inline docs/5.0/pages/enterprise/quickstart-enterprise.md:44]: The Teleport service is called `node`. I would suggest to leave it as `node` here for consistency. We could also consider `ssh node`.

File: docs/5.0/pages/enterprise/quickstart-enterprise.md [hunk 7]
```diff
@@ -39,11 +41,11 @@ the following DNS names and IPs:
...
+_"node.example.com"_  | 10.1.1.11      | This server will only run the SSH service.
```
Comment (travelton) [inline docs/5.0/pages/enterprise/quickstart-enterprise.md:45]: Might be good to tie these words together here? `SSH node service`

File: docs/5.0/pages/enterprise/quickstart-enterprise.md [hunk 8]
```diff
@@ -119,27 +118,64 @@ ssh_service:
...
+$ sudo systemctl status teleport
 ```
+## Teleport Status and Networking
+
+You can review the logs of the Teleport service with `journalctl -fu teleport` and see its current status
+with `sudo systemctl status teleport`.
 
-Teleport daemon should start and you can use `netstat -lptne` to make sure that
-it's listening on [TCP/IP ports](../admin-guide.md#ports). On _auth.example.com_, it should
+You can use `netstat -lptne` to review the port that Teleport is
```
Comment (travelton) [inline docs/5.0/pages/enterprise/quickstart-enterprise.md:177]: Mixed use of sudo and no sudo on these commands. Would it be a good idea to include sudo?

**Review by klizhentas — APPROVED** [review]

**Pull request merged**